src/main/app/feature-flags/FeatureFlags.ts

```diff
@@ -0,0 +1,44 @@
+import { NextFunction, Request, Response } from 'express';
+import { HTTPError } from '../../HttpError';
+import config from 'config';
+
+export class FeatureFlags {
+  constructor(private readonly featureFlagClient: FeatureFlagClient) {
+    this.featureFlagClient = featureFlagClient;
+  }
+
+  public getFlagValue = (flagKey: string, defaultValue = false) => {
+    if(config.has('featureFlags.flags.' + flagKey)) {
+      const value = config.get('featureFlags.flags.' + flagKey);
+      return Promise.resolve(value);
+    }
+
+    return this.featureFlagClient.getFlagValue(flagKey, defaultValue);
+  };
+
+  public getAllFlagValues = (defaultValue = false) => {
+    const localFlags = config.get('featureFlags.flags') as { [key: string]: boolean };
+    return this.featureFlagClient.getAllFlagValues(defaultValue)
+      .then(values => {
+        return { ...values, ...localFlags };
+      });
+  }
+
+  public toggleRoute = (flagKey: string, defaultValue = false) => {
+    return (req: Request, res: Response, next: NextFunction) => {
+      this.getFlagValue(flagKey, defaultValue)
+        .then(value => {
+          value ? next() : next(new HTTPError('FORBIDDEN', 403));
```

Comment:
Think there is an npm package called http-status-codes which have constants for those status codes. So instead of wring 403, is that better to use StatsCodes.FORBIDDEN? Same with the error code 500. That's what I would do for a Java application so I guess this can be applied here too.

Reply:
Ooh actually I did something similar elsewhere in this project by importing the constants from the http2 package, as that provides the same status codes